P202-Training.Domain/P202.Training.Domain.csproj

```diff
@@ -57,7 +57,7 @@
     <Compile Include="Models\Role.cs" />
     <Compile Include="Models\User.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="UsersService.cs" />
+    <Compile Include="RoleService.cs" />
```

Comment:
I think you excluded the UsersService and added the RoleService, is that correct?